### ISSUE TITLE
some workflows cannot be displayed

### ISSUE THREAD

**Issue by huangkun1985** [comfyanonymous/ComfyUI#5742]
### Expected Behavior

show the workflow

### Actual Behavior

when i drag the json file of some workflows into the comfyui, nothing happened. and when i refreshed the page, it shows a blank area. and please note that the logs shows no update.
![01](https://github.com/user-attachments/assets/21f114ee-bbcd-4167-b2e9-e627d51cb857)

### Steps to Reproduce

[workflow-ecommerceinpaintingmay-be-best-for-furniture-pad-imagesegmentanythingultrav2brushnetanyline-controlnetdepth-controlnetimagecompositemaskedimage-blendiclightdetailtransferupscale-qKWDk06An4EjcnjRLD.json](https://github.com/user-attachments/files/17878660/workflow-ecommerceinpaintingmay-be-best-for-furniture-pad-imagesegmentanythingultrav2brushnetanyline-controlnetdepth-controlnetimagecompositemaskedimage-blendiclightdetailtransferupscale-qKWDk06An4EjcnjRLD.json)

### Debug Logs

```powershell
logs will not update
```

**Comment by huangkun1985** [comment]
![02](https://github.com/user-attachments/assets/c9a62394-52ad-4bbe-9ff8-55a7a407e09d)
and if i click a button of the adaptive view, it shows an error.

**Comment by iamrohitanshu** [comment]
I'm getting the same issue on the latest update using the portable comfyui.
After this error, other workflows can't be loaded as well unless the page is refreshed.

**Comment by sh570655308** [comment]
same

**Comment by LukeG89** [comment]
@huangkun1985  I had no problem opening your workflow (even with that extremely long name)

![image](https://github.com/user-attachments/assets/26a09891-06e1-47c6-87a5-40d567e6eb8f)

Have you tried upgrading ComfyUI and all custom nodes?

Have you tried disabling your nodes using the `--disable-all-custom-nodes` argument?
There might be a conflict with one of your nodes.

**Comment by sh570655308** [comment]
@LukeG89 upgrading all custom nodes works for me, thanks

**Comment by iamrohitanshu** [comment]
Updated all the custom nodes via manager, now it works! Thanks @LukeG89 !

**Comment by lucabaronti** [comment]
Same problem, in my case it was the [use-everywhere](https://github.com/chrisgoringe/cg-use-everywhere/issues/247) node

**Comment by chrisgoringe** [comment]
Have you updated? Older versions of UE definitely have this problem.

**Comment by lucabaronti** [comment]
can confirm that manually updating the node solve the problem.

**Comment by xqyd** [comment]
Yes, the problem is with the https://github.com/chrisgoringe/cg-use-everywhere/issues/247 node

**Comment by chrisgoringe** [comment]
@xqyd UE was updated to fix this, but for some reason the manager isn't picking up the update for everyone. I have multiple reports that it is fixed by updating, manually if necessary:

```sh
cd [comfy]/custom_nodes/cg-use-everywhere
git pull
```

**Comment by erleguizamo** [comment]
> UE se actualizó para solucionar esto, pero por alguna razón el administrador no recoge la actualización para todos. Tengo varios informes que se solucionan actualizando, manualmente si es necesario:
> 
> ```shell
> cd [comfy]/custom_nodes/cg-use-everywhere
> git pull
> ```

This worked for me thanks
